src/qt/optionsmodel.cpp

```diff
@@ -282,7 +283,11 @@ QVariant OptionsModel::data(const QModelIndex & index, int role) const
         case CoinControlFeatures:
             return fCoinControlFeatures;
         case DatabaseCache:
-            return settings.value("nDatabaseCache");
+            if (!gArgs.GetArg("-dbcache", "").empty())
+                return (qint64)gArgs.GetArg("-dbcache", nDefaultDbCache);
```

Comment:
Why is this cast necessary?

Reply:
Hm, for the code to compile? Am I missing something?

![deepinscreenshot_select-area_20181021215649](https://user-images.githubusercontent.com/10647624/47311377-6b10dc80-d5ee-11e8-9e39-2b71aeaafc50.png)